src/feature/hs/hs_config.h

```diff
@@ -15,6 +15,14 @@
 #define HS_CONFIG_MAX_STREAMS_PER_RDV_CIRCUIT 65535
 /* Maximum number of intro points per version 3 services. */
 #define HS_CONFIG_V3_MAX_INTRO_POINTS 20
+/* Default value for the introduction DoS defenses. */
```

Comment:
Let's document all of these and specify how each of them is used. For validation or as default values? By the service or the intro point?

Reply:
Well hmmm... you sure that is necessary? We have man page explaining each of them. We have a full proposal explaining what they do.

And there is `_DEFAULT` for each param for the _exact_ purpose of indicating the default value? And MIN/MAX is obviously for validation.

Not sure what more would be useful here without duplicating quite a bit the same documentation?